docs/reference/intro.md

```diff
@@ -6,4 +6,4 @@ There are also reference pages for the Map, Accumulo, Proxy and Federated Gaffer
 
 !!! info
 
-    In general, this documentation covers the most common use cases, it doesn't cover everything. The Javadocs for [Gaffer](https://gchq.github.io/gaffer/) and [Koryphe](https://gchq.github.io/koryphe/) have more specialised information.
\ No newline at end of file
+    In general, this documentation covers the most common use cases, it doesn't cover everything. The Javadocs for [Gaffer](https://gchq.github.io/Gaffer/overview-summary.html) and [Koryphe](https://gchq.github.io/koryphe/) have more specialised information.
```

Comment:
Should koryphe have the `overview-summary.html` added too?

Reply:
That link appears to work just fine unlike the Gaffer one.

Reply:
The Gaffer one works for me as long as it has capital G. To be fair, I believe it has the page name everywhere else